x-pack/filebeat/module/iptables/log/ingest/pipeline.yml

```diff
@@ -210,6 +230,14 @@ processors:
         }
       }
 
+- set:
+    field: event.outcome
+    value: success
+    if: "ctx?.event?.type != null && ctx.event.type.contains('allowed')"
+- set:
+    field: event.outcome
+    value: failure
+    if: "ctx?.event?.type != null && ctx.event.type.contains('denied')"
```

Comment:
I don't think we want to do this.  Just because the network traffic is denied doesn't mean it is a failure.  We would mark it as a failure if we couldn't deny the network traffic.

https://www.elastic.co/guide/en/ecs/current/ecs-using-the-categorization-fields.html#_firewall_blocking_a_network_connection

Reply:
ok so if the network event is from the perspective of the host, it would be failure but for the network device its success? Very thin lines on when to use which.  I will remove it.

Reply:
updated.